src/constants/dummyData.js

```diff
@@ -1,10 +1,158 @@
-// candidates
-// employees
-// jobs
-// notifications
-// onBoarding
-// organizations
-// project onbording
-// birthdays
-// work anniversaries
-// holidays
+export const employees = [
+  {
+    id: '0',
+    name: 'Srikanth',
+    position: 'Executive - US HR',
+    employeeId: '039348',
```

Comment:
Try to keep the keys in object same for employees, once you fix it this PR should be able to merge.

Reply:
Done 👍🏼